pallets/task/src/lib.rs

```diff
@@ -116,7 +120,7 @@ pub mod pallet {
   	pub enum TaskStatus {
     	Created,
     	InProgress,
-		Closed,
+		Accepted,
```

Comment:
May be `Finished` is better name here, from status `Finished` ether it gets accepted or rejected.

Reply:
Or to align batter with event name, we can name it `Completed`

Reply:
Added Completed AND Acceped.